### ISSUE TITLE
[Bug]: Blob core placer gets trapped in AI sat, forced to random place

### ISSUE THREAD

**Issue by Tsar-Salat** [BeeStation/BeeStation-Hornet#9426]
### Describe the bug

I was observing AI when I accepted the dynamic ghost poll for blob, the Blob core placer spawned me in my current location, the AI sat.

The thing is, the majority of the AI sat is near the AI, and thus you cannot place your core.

Whoever designed blob placement also made it so you cannot physically move the placer eyeobj anywhere there isnt a tile, which means I was forced to use the awful random placement, and died.

### What did you expect?

I expected free movement in placing my blob core. Not placing in space is understandable, but space being a barrier to placement on the MAIN station is not.

Wherever I was before I was the antag should have NO bearing on where I get to spawn as a blob.

### Reproducible steps

1. Be observer
2. move to pressurized area with no straight access to the station center
3. get transformed into blob placer
4. be unable to move anywhere, completely defeating the strategy of blob placement
5. cry

**Comment by itsmeow** [comment]
Hmmm.. you should be able to move anywhere prior to placing your core, and then be restricted after placing your core. Aside from space, which is always blocked I think, which might be the problem here.

may be conntected to #8381

**Comment by itsmeow** [comment]
Yeah in that PR I changed it so you cannot move into disallowed areas. I did not consider noncontinuous sections.
